src/style.css

```diff
@@ -0,0 +1,123 @@
+
+body {
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.bar {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+}
+
+.search {
+  display: block;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  padding-left: 62px;
+  outline: none;
+  background: #fff;
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.bar:hover {
+  display: block;
+  margin-top: 20px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
```

Comment:
Better add a transparent border for the default state of bar and change only its color on hover.
Fix all similar cases